tools/log-collector/log_collector.go

```diff
@@ -105,16 +106,16 @@ func (l *LogCollector) getResourceObjects(resourcePath string, resource *apiv1.A
 	listPath := fmt.Sprintf("%s/%s", resourcePath, resource.Name)
 	err = l.disClient.RESTClient().Get().AbsPath(listPath).Do(context.TODO()).Into(&objects)
 	if err != nil {
-		if apierrors.IsNotFound(err) || apierrors.IsForbidden(err) {
-			log.Warnf("%s", err.Error())
-			return objects, nil
-		}
 		if !discovery.IsGroupDiscoveryFailedError(err) {
 			log.Errorf("%s", err.Error())
 			return objects, err
 		}
-		log.Errorf("%s", err.Error())
-		return unstructured.UnstructuredList{}, err
+		if apierrors.IsNotFound(err) || apierrors.IsForbidden(err) {
+			log.Warnf("%s", err.Error())
+			return objects, nil
+		}
+		log.Warnf("%s", err.Error())
```

Comment:
add comment .. why error is being logged at `warning` level and is ignored

Reply:
resolved